specs/fulu/p2p-interface.md

```diff
@@ -204,6 +205,16 @@ The following validations MUST pass before forwarding the `sidecar: DataColumnSi
 
 *Note:* In the `verify_data_column_sidecar_inclusion_proof(sidecar)` check, for all the sidecars of the same block, it verifies against the same set of `kzg_commitments` of the given beacon block. Client can choose to cache the result of the arguments tuple `(sidecar.kzg_commitments, sidecar.kzg_commitments_inclusion_proof, sidecar.signed_block_header)`.
 
+###### Distributed Blob Publishing using blobs retreived from local execution layer client
+
+Similar to the Deneb specification, recent blobs MAY be retrieved by querying the Execution Layer via `engine_getBlobsV2`. Clients will be able to convert the EL blob response into data columns to satisfy block availability requirements.
```

Comment:
Honest nodes SHOULD query `engine_getBlobsV2` as soon as they receive a valid gossip block that contains data, convert the response to data columns, and import the result.